packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/calibrate-operation.ts

```diff
@@ -9,10 +9,12 @@ export interface CalibrationOperationStateCiemss extends BaseState {
 	simulationsInProgress: string[];
 
 	inProgressCalibrationId: string;
+	inProgressBeforeForecastId: string;
```

Comment:
I used slightly different wording for optimize version of this
`inProgressPostForecastId`
Do we care to sync these? Do we care which we sync with?